scripts/generate_rust_analyzer.py

```diff
@@ -65,7 +66,10 @@ def append_crate(display_name, root_module, deps, cfg=[], is_workspace_member=Tr
         [],
         is_proc_macro=True,
     )
-    crates[-1]["proc_macro_dylib_path"] = "rust/libmacros.so"
+    libmacros_path = "rust/libmacros.so"
+    if platform.system() == 'Darwin':
+        libmacros_path = "rust/libmacros.dylib"
```

Comment:
You can also use the output of `rustc --print file-names --crate-name macros --crate-type proc-macro -` (where you immediately close stdin) to get the right filename on all platforms supported by rustc.

Reply:
Not sure how to do that best with python, not the best with python :)

Reply:
Not sure how to do that either. I'm fine with leaving this as is.

Reply:
Got it figured out. No using that